lib/github_api/params_hash.rb

```diff
@@ -9,7 +9,7 @@ class ParamsHash < DelegateClass(Hash)
     include Normalizer
     include MimeType
 
-    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers]
+    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers, :options]
```

Comment:
Do we need the :options key? Usually the options themselves are the options if that even makes sense :smile: